alternator/executor.cc

```diff
@@ -1043,6 +1044,10 @@ future<executor::request_return_type> executor::update_table(client_state& clien
         }
     }
 
+    if (rjson::find(request, "BillingMode")) {
```

Comment:
Nitpick: I think you should call verify_billing_mode unconditionally, because it also checks the case that billingmode is missing and some other attributes are not allowed. But in any case, none of this is supported right now, so it doesn't matter :-)